en/docs/Administer/ProductAdministration/ManagingUsersAndRoles/ManagingUserStores/ConfigurePrimaryUserStore/configuring-a-read-only-ldap-user-store.md

```diff
@@ -19,169 +19,241 @@ Follow the given steps to configure a read-only LDAP/AD as the primary user stor
+
+``` 
+[super_admin]
+username = "admin"
+admin_role = "admin"
+create_admin_account = false
 ```
 
--   **&lt;AddAdmin&gt;:** This should be set to 'False' as it will not be allowed to create users and roles in a read-only user store.
--   **&lt;AdminRole&gt;:** The admin role you enter here should already exist in the read-only user store. Otherwise, you must enter an internal role, which will be saved to the internal database of the system when the system starts the first time.
--   **&lt;AdminUser&gt;:** Since we are configuring a read-only LDAP as the primary user store, the user that should have admin permissions is required to be stored in the user store when you start the system for the first time. For example, say a valid username is AdminSOA. Update the `<AdminUser>` section of your configuration as shown above. You do not have to update the password element as it is already set in the user store.
+-   **create_admin_account:** This should be set to 'False' as it will not be allowed to create users and roles in a read-only user store.
+-   **admin_role:** The admin role you enter here should already exist in the read-only user store. Otherwise, you must enter an internal role, which will be saved to the internal database of the system when the system starts the first time.
+-   **username:** Since we are configuring a read-only LDAP as the primary user store, the user that should have admin permissions is required to be stored in the user store when you start the system for the first time. For example, say a valid username is AdminSOA. Update the `admin_role` section of your configuration as shown above. You do not have to update the password element as it is already set in the user store.
+
 
-For information information about the system administrator user, see [Configuring the System Administrator](https://docs.wso2.com/display/ADMIN44x/Configuring+the+System+Administrator) , and for information on how keystores are used in WSO2 products, see [Using Asymmetric Encryption](https://docs.wso2.com/display/ADMIN44x/Using+Asymmetric+Encryption) .
+If the user store can be written to, you can add the super tenant user to the user store. Therefore, create_admin_account should be set to true as shown below.
+``` 
+[super_admin]
+username = "admin"
+admin_role = "admin"
+create_admin_account = true
+```
+For information information about the system administrator user, see [Configuring the System Administrator](https://apim.docs.wso2.com/en/latest/Reference/ConfigCatalog/#super-admin-configurations) , and for information on how keystores are used in WSO2 products, see [Using Asymmetric Encryption](https://apim.docs.wso2.com/en/latest/Administer/ProductSecurity/General/UsingAsymmetricEncryption/creating-new-keystores/) .
```

Comment:
When adding links use {{base_path}} 

For example:
https://apim.docs.wso2.com/en/latest/Reference/ConfigCatalog/#super-admin-configurations
Should be
{{base_path}}/Reference/ConfigCatalog/#super-admin-configurations

Reply:
Fixed with 491587f